src/test/ReactTestUtils.js

```diff
@@ -168,7 +168,7 @@ var ReactTestUtils = {
       var instClassName = inst.props.className;
       return ReactTestUtils.isDOMComponent(inst) && (
         instClassName &&
-        (' ' + instClassName + ' ').indexOf(' ' + className + ' ') !== -1
+        (' ' + instClassName.replace(/[\r\n]/g, ' ') + ' ').indexOf(' ' + className + ' ') !== -1
```

Comment:
We could probably simplify this entirely and do `('' + instClassName).split(/\s/).indexOf(className)`.